starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemObject.java

```diff
@@ -0,0 +1,48 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by Serkan on 25/10/16.
+ */
+
+public class ItemObject {
```

Comment:
I suggest using name more specific to your app, like "ShoppingListItem" or "GroceryItem" rather than a very generic name like "ItemObject". I know one of the examples in class used a very generic name, but that was just because it had very generic database entries that didn't represent any real-life thing.